packages/utilities/components/percentage.ts

```diff
@@ -0,0 +1,16 @@
+const calculate = (a, b) => {
```

Comment:
wonder why this is an arrow function but `percentage` is a regular function

Reply:
It looks like it's more like a pattern of the helper functions in the PR. Any specific reason?